src/dos/dos_programs.cpp

```diff
@@ -319,19 +317,15 @@ class MOUNT : public Program {
 
 			if (temp_line[temp_line.size()-1]!=CROSS_FILESPLIT) temp_line+=CROSS_FILESPLIT;
 			Bit8u bit8size=(Bit8u) sizes[1];
-			if (type=="cdrom") {
-				int num = -1;
-				cmd->FindInt("-usecd",num,true);
-				// TODO: 
-				// if (num >= 0)
-				//     WriteOut warning, -usecd ignored, no physical CD support
-				int error = 0;
-				if (cmd->FindExist("-ioctl", false)) {
-					WriteOut(MSG_Get("MSCDEX_WARNING_IOCTL"));
-				}
-				if (cmd->FindExist("-noioctl", false)) {
-					WriteOut(MSG_Get("MSCDEX_WARNING_IOCTL"));
+
+			if (type == "cdrom") {
+				// Following options were relevant only for physical CD-ROM  support:
+				for (auto opt : {"-usecd", "-noioctl", "-ioctl", "-ioctl_dx", "-ioctl_mci", "-ioctl_dio"}) {
+					if (cmd->FindExist(opt, false))
```

Comment:
![2019-12-15_20-31](https://user-images.githubusercontent.com/1557255/70879372-ec0cff00-1f79-11ea-88ac-7da526f025ba.png)
I'm not sure if `-cd` can be provided a similar message; altough it's a stand-alone flag given to `mount`, so wouldn't fall under this `cdrom` condition.